proto/sequencers/genesis.proto

```diff
@@ -12,7 +13,14 @@ option go_package = "github.com/dymensionxyz/dymension-rdk/x/sequencers/types";
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
 
+  // the address and its permissions to trigger some special actions
+  // on the rollapp
+  repeated AddressPermissions address_permissions = 2 [
+    (gogoproto.moretags) = "yaml:\"address_permissions\"",
+    (gogoproto.nullable) = false
+  ];
+
   // genesis_operator_address defines the genesis operator address of the
   // sequencer.
-  string genesis_operator_address = 2;
-}
\ No newline at end of file
+  string genesis_operator_address = 3;
```

Comment:
is `sequencer_operator_address` a better name?

Reply:
This one is the original name in sequencer module, I didn't change anything about it. So if it's required, I think we will need another chores issue for that.